code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -285,6 +285,17 @@
 		The idea of a \"rudimentary\" iron wall makes no sense at all! Is anything i'm even saying here true? Someone's gotta fact check this!"
 	icon_state = "stone_floor"
 
+/turf/open/floor/stone/icemoon
+	initial_gas_mix = ICEMOON_DEFAULT_ATMOS
+	name = "stone brick floor"
+	desc = "Odd, really, how it looks exactly like the iron walls yet is stone instead of iron. Now, if that's really more of a complaint about\
```

Comment:
Odd, really, for a moment I had thought this description was different. Now, I believe this is the result of some copypasta.